app/src/main/res/layout/activity_crime.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Hmm, let's see if we can use `FragmentContainerView` mentioned in the tech video from last week
https://www.youtube.com/watch?v=RS1IACnZLy4

Also please check https://proandroiddev.com/android-fragments-fragmentcontainerview-292f393f9ccf